kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/WorkflowRunnerViewModel.kt

```diff
@@ -76,10 +76,12 @@ internal class WorkflowRunnerViewModel<OutputT : Any>(
     }
   }
 
-  private val snapshotJob = GlobalScope.launch(context) {
-    workflowHost.renderingsAndSnapshots
-        .map { it.snapshot }
-        .collect { lastSnapshot = it }
+  init {
+    viewModelScope.launch(context) {
+      workflowHost.renderingsAndSnapshots
+          .map { it.snapshot }
+          .collect { lastSnapshot = it }
+    }
   }
 
   private var workflowJob: Job? = null
```

Comment:
It looks like workflowJob is never set?

Reply:
https://github.com/square/workflow/blob/a6e898ca25ea64e4b3c75a7b92a182830bd35465/kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/WorkflowRunnerViewModel.kt#L126

Reply:
But that's going away in #447.